src/unix/fsevents.c

```diff
@@ -641,17 +641,12 @@ static int uv__fsevents_loop_init(uv_loop_t* loop) {
     goto fail_signal_source_create;
   }
 
-  /* In the unlikely event that pthread_attr_init() fails, create the thread
-   * with the default stack size. We'll use a little more address space but
-   * that in itself is not a fatal error.
-   */
   attr = &attr_storage;
   if (pthread_attr_init(attr))
-    attr = NULL;
+    abort();
```

Comment:
You can go a bit further here: delete `attr_storage` above, change the type of `attr` to `pthread_attr_t`, and change the `pthread_attr_destroy()` call below to:
```c
if (pthread_attr_destroy(&attr))
  abort();
```
LGTM apart from that.

Reply:
Good catch, thank you!